Frends.Salesforce.CreateSObject/Frends.Salesforce.CreateSObject/Definitions/Result.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Frends.Salesforce.CreateSObject.Tests")]
+namespace Frends.Salesforce.CreateSObject
```

Comment:
Please use one-line namespace definition. Also, the namespace should be `Frends.Salesforce.CreateSObject.Definitions`.